ExerciseOne/exercise-one.html

```diff
@@ -3,6 +3,65 @@
     <script type="text/javascript">
         console.log("javascript code goes here")
     </script>
+    <style>
```

Comment:
So this is great.  The `<style>` tag always belong in the `<head>`, however, you left a copy of the `<style>` tag and its selectors at the bottom of the document.  This would need to be removed.